Mono.Cecil.Tests.props

```diff
@@ -2,9 +2,18 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildDirectory>$(MSBuildProjectDirectory)</BuildDirectory>
+    <IsTestProject>true</IsTestProject>
   </PropertyGroup>
   <Import Project="Mono.Cecil.props" />
-  <ItemGroup>
+  <PropertyGroup Condition="'$(NetStandard)' == 'true'">
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Why do we need binding redirects?

Reply:
In case we cross-target tests to also run on .NET Framework.

Reply:
@tmat so we can remove this and the workaround below?